weed/s3api/s3_constants/extend_key.go

```diff
@@ -1,6 +1,7 @@
 package s3_constants
 
 const (
-	ExtAcpKey       = "Seaweed-X-Amz-Acp"
+	ExtAmzOwnerKey  = "Seaweed-X-Amz-Owner"
+	ExtAmzAclKey    = "Seaweed-X-Amz-Acl"
```

Comment:
Here, the owner and grants of acp are divided into two key storages, and the owner only stores the accountId (string). When verifying the object acl, the owner's check and the grants are checked separately. For the case that the owner check fails, There is no need to validate grants, thus avoiding unnecessary `json.Unmarshal(grants)` calls

eg:
```go
	//owner access
	objectOwner := getAcpOwner(entry, *bucketMetadata.Owner.ID)
	if accountId == objectOwner {
		return s3err.ErrNone
	}

	//find in Grants
	acpGrants := getAcpGrants(entry)
	if acpGrants != nil {
		reqGrants := determineReqGrants(accountId, s3_constants.PermissionRead)
		for _, requiredGrant := range reqGrants {
			for _, grant := range acpGrants {
				if reflect.DeepEqual(requiredGrant, grant) {
					return s3err.ErrNone
				}
			}
		}
	}
```